joomla/joomla-cms
# Add registration menu to login module

This PR adds an option to login module to set a menu item for registration page.
Useful to override default registration link "index.php?option=com_users&view=registration" to a custom page

### Summary of Changes
New option in login module


### Testing Instructions
Apply the PR.
1) Go to front end: nothing should be changed in registration link in login module.
2) Edit login module and set a menu item: go to front-end and registration link should point to the deisderd menu item.

### Expected result
See above.


### Actual result



### Documentation Changes Required
